pkg/chunked/storage_linux_test.go

```diff
@@ -211,3 +211,39 @@ func TestTypeToOsMode(t *testing.T) {
 		})
 	}
 }
+
+func TestCleanAbsDirectory(t *testing.T) {
+	tests := []struct {
+		path     string
+		expected string
+	}{
+		{"", "/"},
```

Comment:
(Absolutely non-blocking: Consider adding entries for `"."`, `".."`, and maybe `"foo/../.."`,`"/foo/../.."`)